google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/TableInfo.java

```diff
@@ -214,6 +227,17 @@ static class BuilderImpl extends Builder {
       if (tablePb.getCloneDefinition() != null) {
         this.cloneDefinition = CloneDefinition.fromPb(tablePb.getCloneDefinition());
       }
+      if (tablePb.getTableConstraints() != null) {
```

Comment:
Why was the TableConstraints abstraction excluded in the Table class implementation?